data/json/snippets/grave_labels.json

```
@@ -31,7 +31,50 @@
       "I held you your whole life",
       "So small, so sweet, so soon",
       "Ours for a little while",
-      "You died for us, we'll live for you"
+      "You died for us, we'll live for you",
+      "See you in hell",
+      "You deserved it",
+      "You got what was coming to you",
+      "<nothing is written here>",
+      "I hope you rot",
+      "I hope you burn",
+      "Dying was too good an end for you",
+      "I told you I'd put you in the dirt",
+      "Wish I could have done it",
+      "Here lies a traitor",
+      "Here lies a scoundrel",
+      "SCUM",
+      "LIAR",
+      "COWARD",
+      "I never liked you",
+      "We never liked you",
+      "Don't come back",
+      "Stay in the ground you sack of shit",
+      "<several slurs and insults from varying handwritings>",
+      "YOU GAVE ME ADES YOU SHIT",
+      "Murderer",
+      "Thief",
+      "Fuck you",
+      "Took the coward's way out",
+      "You're better this way",
+      "Enjoy your eternal rest, you BITCH",
+      "Good riddance",
+      "37:13",
+      "Waste of a perfectly good bullet",
+      "As much as the world is hell currently, your death made it that much closer to heaven",
+      "Shouldn't have stolen from us",
+      "Now I can rest easy",
+      "Now we can rest easy",
+      "<a crass drawing>",
+      "Even the zeds would make a better comrade than you",
+      "To think, you were the one to leave me for dead",
+      "Didn't have to end like this",
+      "Hope even the worms will touch your corpse",
```

```suggestion
      "Hope even the won't will touch your corpse",
```
This seems to make better sense to me.